api/share/tools.py

```diff
@@ -0,0 +1,28 @@
+from typing import Dict, List, Union
+from http import HTTPStatus
+
+
+def build_correct_http_response(data: Union[Dict, List],
+                                current_url: str,
+                                status: HTTPStatus,
+                                errors: List = None
```

Comment:
```suggestion
                                errors: list = None
```
`list[something]` would be nice

Reply:
the same

Reply:
Ok, I got it